docs/source/glossary.rst

```diff
@@ -17,6 +17,9 @@ Glossary of all experiment attributes with short definitions.
   Fractional factorial:
     A design that is the fraction of a full-factorial design.
     A full-factorial :math:`2^{k}` contains all combinations of the levels of :math:`k` factors.
+    
+  Full factorial:
+    A full-factorial :math:`2^{k}` contains all combinations of the levels of :math:`k` factors. 
```

Comment:
some adaptations are needed here after merging the PR for checking the documented keywords